src/api/client/Claims.ts

```diff
@@ -483,4 +503,77 @@ export class Claims {
    *  - a custom claim type with the same `name` already exists
    */
   public registerCustomClaimType: ProcedureMethod<RegisterCustomClaimTypeParams, BigNumber>;
+
+  /** Retrieves a custom claim type based on the provided options.
+   *
+   * @remarks
+   * This method can retrieve a custom claim type using either its `name` or `id`.
+   *
+   * @param opts - The options object used for retrieving the custom claim type.
+   * @param opts.name - The name of the custom claim type to retrieve.
+   * @param opts.id - The ID of the custom claim type to retrieve.
+   *
+   * @throws {PolymeshError}
+   * Throws a PolymeshError if neither `name` nor `id` is provided in the options.
+   *
+   * @returns A promise that resolves to the `CustomClaimTypeToHuman` object if found, or null otherwise.
+   */
+  public async getCustomClaimType(
```

Comment:
Personally I think exposing `getCustomClaimTypeByName` and `getCustomClaimTypeById` is a more intuitive API.